### PR TITLE
Implement SSO + WebApp scenario

### PR DESCRIPTION
**Pull request by Sgitario**
This is the same example as in SSO + Keycloak, but using a webapp application (`quarkus.oidc.application-type=web-app`).

### CODE REVIEW

**Comment by Sgitario** [comment]
The job for jvm worked fine: https://quarkus-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/openshift-test-suite-ci/45

**Comment by rsvoboda** [comment]
@Sgitario please rebase

**Comment by Sgitario** [comment]
> @Sgitario please rebase

Done.

File: security/keycloak-webapp/src/main/docker/Dockerfile.jvm
Comment (rsvoboda) [inline security/keycloak-webapp/src/main/docker/Dockerfile.jvm:1]: Both docker files are just copy and paste from http module I guess.

They are quite legacy ones, current Docker files are based on `registry.access.redhat.com/ubi8/ubi-minimal:8.1`.
Check https://quarkus.io/guides/getting-started#bootstrapping-the-project generated project to see the latest version of docker files.

File: security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java
Comment (rsvoboda) [inline security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java:24]: Create issue for this TODO

File: security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/containers/KeycloakQuarkusTestResource.java
Comment (rsvoboda) [inline security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/containers/KeycloakQuarkusTestResource.java:23]: `11.0.3` is the latest one

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Few small changes, otherwise good!

File: security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java
Comment (Sgitario) [reply security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java:24]: Done: https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/127

File: security/keycloak-webapp/src/main/docker/Dockerfile.jvm
Comment (Sgitario) [reply security/keycloak-webapp/src/main/docker/Dockerfile.jvm:1]: Updated both files.

File: security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/containers/KeycloakQuarkusTestResource.java
Comment (Sgitario) [reply security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/containers/KeycloakQuarkusTestResource.java]: Updated and checked that the test is still working fine.

**Review by rsvoboda — APPROVED** [review]

**Comment by rsvoboda** [comment]
Waiting for CI before merging

File: security/keycloak-webapp/src/main/docker/Dockerfile.jvm
Comment (Ladicek) [reply security/keycloak-webapp/src/main/docker/Dockerfile.jvm:1]: We should probably remove all the `src/main/docker` directories. I originally left them there, in hope that we'll eventually use them, but that doesn't seem to be the case.

Comment (rsvoboda) [reply security/keycloak-webapp/src/main/docker/Dockerfile.jvm:1]: Depends what "Docker based build for OpenShift" will translate into.
So I would keep them in place for few days/weeks before we know for sure the is no use for them.

**Pull request merged**
